blueflood-elasticsearch/src/main/java/com/rackspacecloud/blueflood/io/Discovery.java

```diff
@@ -24,6 +24,9 @@
 import java.util.Map;
 
 public class Discovery {
+
+    protected final static String SEPARATOR = ":";
```

Comment:
Does this separator have to be the same as Token.SEPARATOR ? If yes, then do we need to just place it in one class?